torch/_dynamo/convert_frame.py

```diff
@@ -358,6 +358,7 @@ def profile_wrapper(*args: _P.args, **kwargs: _P.kwargs) -> _T:
             prof.dump_stats(profile_path)
         except PermissionError:
             log.warning("Cannot write to %s", str(profile_path))
```

Comment:
While you are editing here, might as well print exception info too?
```suggestion
            log.warning("Cannot write to %s", str(profile_path), exc_info=True)
```